optimum/habana/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -300,58 +369,84 @@ def gaudi_wav2vec2_encoder_forward(
     )
 
 
-def gaudi_wav2vec2_forward(
+_HIDDEN_STATES_START_POSITION = 2
+
+
+def gaudi_wav2vec2forctc_forward(
     self,
     input_values: Optional[torch.Tensor],
     attention_mask: Optional[torch.Tensor] = None,
-    mask_time_indices: Optional[torch.FloatTensor] = None,
     output_attentions: Optional[bool] = None,
     output_hidden_states: Optional[bool] = None,
     return_dict: Optional[bool] = None,
-) -> Union[Tuple, Wav2Vec2BaseModelOutput]:
+    labels: Optional[torch.Tensor] = None,
+) -> Union[Tuple, CausalLMOutput]:
+    r"""
+    labels (`torch.LongTensor` of shape `(batch_size, target_length)`, *optional*):
+        Labels for connectionist temporal classification. Note that `target_length` has to be smaller or equal to
+        the sequence length of the output logits. Indices are selected in `[-100, 0, ..., config.vocab_size - 1]`.
+        All labels set to `-100` are ignored (masked), the loss is only computed for labels in `[0, ...,
+        config.vocab_size - 1]`.
```

Comment:
No need to keep this since it is exactly the same as in Transformers

Reply:
Ok, will make the change.